src/helpformaction.cpp

```diff
@@ -74,8 +74,7 @@ void HelpFormAction::prepare()
 			fmt.do_format(cfg->get_configvalue("help-title-format"),
 				width));
 
-		const auto descs = v->get_keymap()->get_keymap_descriptions(
-				v->get_keymap()->get_flag_from_context(context));
```

Comment:
After this change, do we even need `get_flag_from_context()` anymore?

Reply:
Yes, it is still used in the `get_keymap_description()` function itself: https://github.com/newsboat/newsboat/blob/606b84b0e216094efaa80d24c49ca17f594a59e2/src/keymap.cpp#L518

The flag is needed to determine if an operation is supported in the specified context.

Reply:
Oh. I'm still reviewing that part; should've held the comments off until then. Thanks for the explanation! :+1: